h/static/scripts/annotator/monkey.coffee

```diff
@@ -9,6 +9,12 @@ Util = Annotator.Util
 delete Annotator.prototype.events[".annotator-hl mouseover"]
 delete Annotator.prototype.events[".annotator-hl mouseout"]
 
+# Add createHighlight
+Annotator.prototype.createHighlight = ->
+  annotation = {$highlight: true}
```

Comment:
Furthermore, we have a function to test whether or not an annotation is a highlight -- see [here](https://github.com/hypothesis/h/blob/56913583b510d89b1162691666a56eaae21fffbc/h/static/scripts/directives/annotation.coffee#L61).
What's the point of setting a dedicated flag for it?

Especially since annotations can *become* highlights when we remove the body (or tags), and can *stop* being highlights when adding body (or tags). Having a flag to update accordingly complicates things.

I propose using the test function inside, and getting rid of this flag.